src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/page/SendOrReply.java

```diff
@@ -37,7 +38,8 @@ public void messageBuilder(PageBuilder pageBuilder,String condition) {
             .label("replyMessageLab", "## Reply to message","messageAction=\"replyMessage\"")
             .complex(CaseData::getMessageSendDetails)
             .mandatory(MessageSendDetails::getMessageReceiverRoles)
-            .mandatory(MessageSendDetails::getMessageReasonList)
+            .mandatory(MessageSendDetails::getMessageReasonList, "loggedInUserRole=\"caseworker\"")
```

Comment:
I think this list is not specific to 'caseworker' role, its for any non-judicial' role